src/test/java/uk/gov/companieshouse/psc/delta/mapper/PscMapperTest.java

```diff
@@ -0,0 +1,255 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.util.FileCopyUtils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import uk.gov.companieshouse.api.delta.NameElements;
+import uk.gov.companieshouse.api.delta.Psc;
+import uk.gov.companieshouse.api.delta.PscDelta;
+import uk.gov.companieshouse.api.psc.Address;
+import uk.gov.companieshouse.api.psc.Data;
+import uk.gov.companieshouse.api.psc.DateOfBirth;
+import uk.gov.companieshouse.api.psc.ExternalData;
+import uk.gov.companieshouse.api.psc.FullRecordCompanyPSCApi;
+import uk.gov.companieshouse.api.psc.ItemLinkTypes;
+import uk.gov.companieshouse.api.psc.SensitiveData;
+import uk.gov.companieshouse.api.psc.UsualResidentialAddress;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {
+        PscMapperImpl.class})
+public class PscMapperTest {
+
+    private ObjectMapper mapper;
+    private PscDelta individualDeltaObject;
+    private PscDelta corporateDeltaObject;
+    private PscDelta superSecureDeltaObject;
+    private Psc individualPsc;
+    private Psc corporatePsc;
+    private Psc superSecurePsc;
+    private Address serviceAddress;
+    private UsualResidentialAddress usualResidentialAddress;
+    private NameElements nameElements;
+    private DateOfBirth dateOfBirth;
+
+    @Autowired
+    PscMapper pscMapper;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        mapper = new ObjectMapper();
+
+        String individualPath = "individual-psc-delta-example.json";
+        String corpPath = "corporate-entity-psc-delta-example.json";
+        String superSecurePath = "super-secure-psc-delta-example.json";
+
+        String individualInput = FileCopyUtils.copyToString(new InputStreamReader(
+                ClassLoader.getSystemClassLoader().getResourceAsStream(individualPath)));
+        String corpInput = FileCopyUtils.copyToString(new InputStreamReader(
+                ClassLoader.getSystemClassLoader().getResourceAsStream(corpPath)));
+        String superSecureInput = FileCopyUtils.copyToString(new InputStreamReader(
+                ClassLoader.getSystemClassLoader().getResourceAsStream(superSecurePath)));
+
+        individualDeltaObject = mapper.readValue(individualInput, PscDelta.class);
+        individualPsc = individualDeltaObject.getPscs().get(0);
+
+        corporateDeltaObject = mapper.readValue(corpInput, PscDelta.class);
+        corporatePsc = corporateDeltaObject.getPscs().get(0);
+
+        superSecureDeltaObject = mapper.readValue(superSecureInput, PscDelta.class);
+        superSecurePsc = superSecureDeltaObject.getPscs().get(0);
```

Comment:
I think these would be a cleaner read if you created a method that accepted a path and returned a PscDelta object. Would also remove the repeated code.